core/templates/pages/admin-page/navbar/admin-navbar.directive.html

```diff
@@ -155,11 +154,11 @@
         </li>
         <li>
           <ul class="nav oppia-navbar-tabs-admin">
-            <li uib-dropdown class="nav-item uib-dropdown float-right">
+            <li uib-dropdown class="nav-item uib-dropdown float-right" ng-class="{'open' : $ctrl.profileDropdownIsActive}">
               <a uib-dropdown-toggle class="nav-link uib-dropdown-toggle oppia-navbar-dropdown-toggle"
                  data-toggle="dropdown"
-                 ng-mouseover="$ctrl.onMouseoverProfilePictureOrDropdown($event)"
-                 ng-mouseleave="$ctrl.onMouseoutProfilePictureOrDropdown($event)">
+                 ng-mouseenter="$ctrl.profileDropdownIsActive = true"
+                 ng-mouseleave="$ctrl.profileDropdownIsActive = false">
```

Comment:
This looks like a duplicate code, can you please create two functions `activateProfileDropdown` and deactivateProfileDropdown` and use them here and elsewhere?

Reply:
done!